lib/Swagger/src/TemplateClassification/Storage/Api/TCSApi.php

```diff
@@ -369,73 +353,67 @@ public function getTemplateTypesOnWikiWithHttpInfo($wiki_id)
         if (isset($_tempBody)) {
             $httpBody = $_tempBody; // $_tempBody is the method argument, if present
         } elseif (count($formParams) > 0) {
             $httpBody = $formParams; // for HTTP post (form)
         }
-        
         // make the API Call
         try {
             list($response, $statusCode, $httpHeader) = $this->apiClient->callApi(
-                $resourcePath, 'GET',
-                $queryParams, $httpBody,
-                $headerParams, '\Swagger\Client\TemplateClassification\Storage\Models\TemplateTypeHolder[]'
+                $resourcePath,
+                'GET',
+                $queryParams,
+                $httpBody,
+                $headerParams,
+                '\Swagger\Client\TemplateClassification\Storage\Models\TemplateTypeHolder[]',
+                '/{wiki_id}'
             );
-            
-            if (!$response) {
-                return array(null, $statusCode, $httpHeader);
-            }
 
-            return array(\Swagger\Client\ObjectSerializer::deserialize($response, '\Swagger\Client\TemplateClassification\Storage\Models\TemplateTypeHolder[]', $httpHeader), $statusCode, $httpHeader);
-            
+            return [$this->apiClient->getSerializer()->deserialize($response, '\Swagger\Client\TemplateClassification\Storage\Models\TemplateTypeHolder[]', $httpHeader), $statusCode, $httpHeader];
         } catch (ApiException $e) {
-            switch ($e->getCode()) { 
-            case 200:
-                $data = \Swagger\Client\ObjectSerializer::deserialize($e->getResponseBody(), '\Swagger\Client\TemplateClassification\Storage\Models\TemplateTypeHolder[]', $e->getResponseHeaders());
-                $e->setResponseObject($data);
-                break;
+            switch ($e->getCode()) {
+                case 200:
+                    $data = $this->apiClient->getSerializer()->deserialize($e->getResponseBody(), '\Swagger\Client\TemplateClassification\Storage\Models\TemplateTypeHolder[]', $e->getResponseHeaders());
+                    $e->setResponseObject($data);
+                    break;
             }
-  
+
             throw $e;
         }
     }
-    
+
     /**
-     * insertTemplateDetails
+     * Operation insertTemplateDetails
      *
      * Save template type data
      *
      * @param int $wiki_id Wikia ID (required)
      * @param int $page_id Article ID (required)
      * @param \Swagger\Client\TemplateClassification\Storage\Models\TemplateTypeProvider $body Provider data (optional)
-     * @return void
      * @throws \Swagger\Client\ApiException on non-2xx response
+     * @return void
      */
     public function insertTemplateDetails($wiki_id, $page_id, $body = null)
     {
-        list($response, $statusCode, $httpHeader) = $this->insertTemplateDetailsWithHttpInfo ($wiki_id, $page_id, $body);
-        return $response; 
+        list($response) = $this->insertTemplateDetailsWithHttpInfo($wiki_id, $page_id, $body);
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Immediately return this expression instead of assigning it to the temporary variable "$response". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1488)